src/module/generate-swagger-spec.ts

```diff
@@ -14,6 +14,13 @@ export interface RoutesConfigRelatedToSwagger {
   controllerPathGlobs?: RoutesConfig['controllerPathGlobs'];
 }
 
+export const getSwaggerOutputPath = (swaggerConfig: SwaggerConfig) => {
+  const ext = swaggerConfig.yaml ? 'yaml' : 'json';
+  const outputBasename = swaggerConfig.outputBasename || 'swagger';
```

Comment:
```suggestion
  const outputBasename = swaggerConfig.outputBasename || 'openapi';
```

Reply:
I take it you're gearing up for a major-version release to go with a change of default value?

Reply:
I am, I'd like to ship this in 3.x